docs/troubleshooting-guide/filters.md

```diff
@@ -0,0 +1,93 @@
+
+To learn more about these features, please read the articles listed below. If you have problems or encounter unexpected behavior, please look at the specific problems and their solutions.
+
+- "[Cross-filtering: using a chart to update a dashboard filter][learn-cross-filtering]"
+- "[Field Filters: create smart filter widgets for SQL questions][learn-field-filters]"
+- "[SQL variables][learn-filter-charts-sql-variables]"
+- "[Adding filters to dashboards with SQL questions][learn-filter-sql-questions]"
+- "[Create filter widgets for charts using SQL variables][learn-filter-sql-variables]"
+- "[Linking filters in dashboards][learn-linking]"
+
+## My filters don't work after a change to the underlying database schema
+
+**How to detect this:** Someone has renamed a column in the database, and a filter that used that column no longer seems to have any effect, or seems to eliminate all of the rows even after you [re-sync][sync-scan] Metabase with the database.
+
+**How to fix this:** Tools like Excel can automatically update formulas when cell values are moved around in a spreadsheet, but databases schemas are much more complicated, so Metabase does not automatically update the names of columns used in questions when the underlying database changes. If the schema has changed, you must update your questions and filters to match.
+
+## My cards are empty when I apply linked filters
+
+**How to detect this:** Some data shows up when the first filter is applied, but nothing shows up when the second (dependent) filter is applied as well. For example, you may have a "State" and a "City" filter linked so that the "City" filter only shows cities in the selected state; some rows are displayed when you select a state, but none appear when you also add a city.
+
+**How to fix this:** The root cause of this problem is usually that there really are no rows that satisfy both conditions. For example, if you manually enter the name of a city that isn't in the selected state, no record will satisfy both conditions. You can check this by writing a native SQL query that you think should produce the same result. If it does not, check for typing mistakes and that you are using the correct type of join.
+
+## My linked filter seems to have no effect
+
+**How to detect this:** After creating and applying a linked filter, you see the same set of rows that you saw before the second (linked) filter was applied.
+
+**How to fix this:** This is the inverse of the previous problem: all of the rows that pass the first test also pass the second. For example, if you create a date filter and then link a category filter to it, the second filter might have no effect. Again, you can check this by writing a native SQL query that you think should produce the reduced result; if it does not, the problem is most likely with the logic.
+
+FIXME: https://github.com/metabase/metabase/issues/14595
+
+## My linked filter widget does not display a dropdown of filtered values
```

Comment:
Perhaps there should also be a section about non-linked filters doesn't show dropdown, which should then tell to change the field type in Data Model (and some reference the the sync troubleshoot).

Reply:
added